omod/src/main/webapp/resources/scripts/field/registerPersonRelationship.js

```diff
@@ -0,0 +1,43 @@
+function RegisterPatientRelationship(patientRelationship) {
+
+  function getInputElementFor(fieldId) {
+    // this is to handle integration with HFE, when the id is set on the parent span,
+    // not the input element itself
+    return $('#' + fieldId).is('input') ?
+      $('#' + fieldId) :
+      $('#' + fieldId).find('input');
+  }
+
+  function getValue(fieldId) {
+    return getInputElementFor(fieldId).val();
+  }
+  function setValue(fieldId, value) {
+    getInputElementFor(fieldId).val(value);
+  }
+
+  patientRelationship.container.find(".searchablePerson").autocomplete({
+        autoFocus: true,
+        source: function (request, response) {
+          if (request.term.length < 3) {
+            return;
+          }
+          let url = '/' + OPENMRS_CONTEXT_PATH + '/ws/rest/v1/person/';
+          $.getJSON(url, {
+            limit: 10,
+            q: request.term,
+            v: 'custom:(id,uuid,display,gender,age)'
+          }, function (result) {
```

Comment:
It looks to me like you are configuring this widget with a `gender` property, but I don't see where you are actually limiting the results to persons that match this gender.  Shouldn't that be done in a filter here?